samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dying-armstrong",
+   "metadata": {},
+   "source": [
+    "In the cell above increase the `nQubits` parameter (by multiples of `nClusters` parameter) such that the simulation becomes evidently slow (e.g. `nQubits`: 8, 10, 12, 14, ...; `nClusters`: 2). On some laptops the simulation becomes slow at `nQubits=20`. Don't make large steps otherwise the simulation can run too long.\n",
+    "After the number of qubits exceeds 15 the state hystogram will not be displayed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "mediterranean-woman",
+   "metadata": {},
+   "source": [
+    "Now let's run our Q# operation on the sparse simulator, whish is done with the [%simulate_sparse](https://docs.microsoft.com/en-us/qsharp/api/iqsharp-magic/simulate_sparse) magic command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "empty-commonwealth",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Let's run a big simulation!\n",
+      "Allocating 200 qubits...\n",
+      "Finished allocating!\n",
+      "Creating 2 clusters of GHZ states between 100 qubits...\n",
+      "Finished cluster 0.\n",
+      "Finished cluster 1.\n",
+      "Finished GHZ clusters!\n",
+      "Collapse and show final state:\n",
+      "[One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero]\n",
+      "Cleaning up...\n",
+      "Done!\n"
+     ]
+    },
+    {
+     "data": {
+      "application/x-qsharp-data": "{\"@type\":\"tuple\"}",
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%simulate_sparse PrepareAndMeasureGhzState nQubits=200 nClusters=2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "uniform-lesson",
+   "metadata": {},
+   "source": [
+    "From the hystogram above you can see that the sparse simulator stores the non-zero amplitudes only, as opposed to the full state simulator.\n",
```

Comment:
```suggestion
    "From the histogram above you can see that the sparse simulator only stores non-zero amplitudes, as opposed to the full-state simulator.\n",
```

Technically, this isn't _quite_ true, since the visualization functionality in IQ# allows truncating out amplitudes that are close to zero when printing from the dense simulator as well. It may be good to clarify slightly.